### PR TITLE
Add get_trace to network analyzer.

### LINKED ISSUE #400: Add get_trace to network analyzer server
A setting that retrives the trace on the screen and does nothing else.

### PR DESCRIPTION
**Pull request by DanielSank**
The new setting just gets a trace from the network analyzer screen and returns it.

Fixes #400

### CODE REVIEW

**Comment by DanielSank** [comment]
@thchwhite

File: PNA/agilentN5242A.py [hunk 1]
```diff
@@ -272,7 +274,8 @@ def freq_sweep(self, c, log=False):
         if sweeptime > 1:
             sweeptime *= self.sweepFactor(c)
             print "sweeptime = ", sweeptime
-            yield util.wakeupCall(sweeptime)  #needs factor of 2 since it runs both forward and backward 
+            # needs factor of 2 since it runs both forward and backward
```
Comment (ejeffrey) [inline PNA/agilentN5242A.py]: What factor of two is this?  I know you are just reformatting, but this comment makes no sense and should be removed or fixed.

File: PNA/agilentN5242A.py [hunk 2]
```diff
@@ -304,7 +326,8 @@ def freq_sweep_phase(self, c, log=False):
         sweeptime, npoints = yield self.startSweep(dev, sweepType)
         if sweeptime > 1:
             sweeptime *= self.sweepFactor(c)
-            yield util.wakeupCall(2*sweeptime)  #needs factor of 2 since it runs both forward and backward 
+            # needs factor of 2 since it runs both forward and backward
```
Comment (ejeffrey) [inline PNA/agilentN5242A.py]: OK, at least there is a factor of 2 here.

File: PNA/agilentN5242A.py [hunk 3]
```diff
@@ -285,8 +288,27 @@ def freq_sweep(self, c, log=False):
         # wait for sweep to finish
         sparams = yield self.getSweepData(dev, c['meas'])
         returnValue((freq*units.Hz, sparams))
-        
-        
+
+    @setting(102, returns='*v[Hz]*c')
+    def get_trace(self, c, trace='S21'):
+        """Get whatever trace is on the screen.
```
Comment (ejeffrey) [inline PNA/agilentN5242A.py]: This seems weird.  The docstring says "get whatever trace is on the screen", but you have to specify an S parameter as an argument.  What happens if S21 is not on the screen?  Why do I have to know what is on the screen to grab it?

File: PNA/agilentN5242A.py (hunk 1 above)
Comment (DanielSank) [inline PNA/agilentN5242A.py]: I have absolutely no idea what this is.

File: PNA/agilentN5242A.py (hunk 3 above)
Comment (DanielSank) [inline PNA/agilentN5242A.py]: I don't know what happens if you have e.g. trace S21 off and try to get it through this setting. What I meant to convey with "on the screen" is "whatever frequency/power/etc conditions are already set up (and you might need to have the trace you want activated, but I'm not sure)".

**Comment by DanielSank** [comment]
@ejeffrey I removed that wacky comment. Ok with the rest?

File: PNA/agilentN5242A.py (hunk 2 above)
Comment (DanielSank) [inline PNA/agilentN5242A.py]: I didn't write these comments. I think removing them is out of bounds of this PR.

**Comment by DanielSank** [comment]
@ejeffrey could you please review this?

**Pull request merged**